docs/spec.md

```diff
@@ -5448,7 +5529,8 @@ More formally, a **StableHLO process** is a combination of:
 already executed), and 3) intermediate values that the process is working on.
 The process starts with input values to the `main` function, progresses through
 the graph of ops updating operation statuses and intermediate values and
-finishes with output values. Further formalization is TBD.
+finishes with output values. Further formalization is TBD
+([#484](https://github.com/openxla/stablehlo/issues/484)).
```

Comment:
#484 not in frontend contract

Reply:
Same as [above](https://github.com/openxla/stablehlo/pull/1076#discussion_r1099053266).